OctoPod/File UI/FilesTreeViewController.swift

```diff
@@ -1,44 +1,45 @@
 import UIKit
 
 class FilesTreeViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UIPopoverPresentationControllerDelegate, DefaultPrinterManagerDelegate, UISearchBarDelegate {
-    
     private var currentTheme: Theme.ThemeChoice!
 
-    let printerManager: PrinterManager = { return (UIApplication.shared.delegate as! AppDelegate).printerManager! }()
-    let octoprintClient: OctoPrintClient = { return (UIApplication.shared.delegate as! AppDelegate).octoprintClient }()
-    let appConfiguration: AppConfiguration = { return (UIApplication.shared.delegate as! AppDelegate).appConfiguration }()
-    let defaultPrinterManager: DefaultPrinterManager = { return (UIApplication.shared.delegate as! AppDelegate).defaultPrinterManager }()
+    let printerManager: PrinterManager = (UIApplication.shared.delegate as! AppDelegate).printerManager!
+    let octoprintClient: OctoPrintClient = (UIApplication.shared.delegate as! AppDelegate).octoprintClient
+    let appConfiguration: AppConfiguration = (UIApplication.shared.delegate as! AppDelegate).appConfiguration
+    let defaultPrinterManager: DefaultPrinterManager = (UIApplication.shared.delegate as! AppDelegate).defaultPrinterManager
 
-    @IBOutlet weak var tableView: UITableView!
-    @IBOutlet weak var searchBar: UISearchBar!
-    @IBOutlet weak var sortByTextLabel: UILabel!
-    @IBOutlet weak var sortByControl: UISegmentedControl!
-    @IBOutlet weak var refreshSDButton: UIButton!
+    @IBOutlet var tableView: UITableView!
```

Comment:
Was the removal of the 'weak' keyword intentional? If so what's the reason for this change? Same for the following 4 instance variables.